src/main/kotlin/app/revanced/patches/reddit/customclients/relayforreddit/api/patch/ChangeClientIdPatch.kt

```diff
@@ -0,0 +1,44 @@
+package app.revanced.patches.reddit.customclients.relayforreddit.api.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.*
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.annotation.Package
+import app.revanced.patcher.extensions.InstructionExtensions.replaceInstruction
+import app.revanced.patcher.patch.OptionsContainer
+import app.revanced.patcher.patch.PatchOption
+import app.revanced.patcher.patch.PatchResultError
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.reddit.customclients.relayforreddit.api.fingerprints.GetClientIdFingerprint
+
+@Patch
+@Name("change-client-id")
+@Description("Changes the OAuth client ID")
+@Compatibility([Package("free.reddit.news"), Package("reddit.news")])
+@Version("0.0.1")
+class ChangeClientIdPatch :
```

Comment:
Extend from `AbstractClientIdPatch`, follow the same style as the existing Reddit client patches.